styles/mixins/spacing.js

```diff
@@ -14,8 +14,22 @@ module.exports = function (config) {
     return breakpointHelper(styles, config);
   };
 
-
+  const axials = {
```

Comment:
Yeah I think it makes more sense that way, the outer mixin call is really just attaching a css node, it shouldn't care how many children that node has.